router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
Do we just hard fail on an empty DEFAULT_HTTPS_PORT? I feel like that's the safest / most predictable response. Let me know soon if you'd prefer otherwise.